examples/with-react-i18next/README.md

```diff
@@ -16,7 +16,7 @@ yarn create next-app --example with-react-i18next with-react-i18next-app
 
 ### Download manually
 
-Download the example:
+Download the example [or clone the repo](https://github.com/zeit/next.js):
```

Comment:
We shouldn't recommend cloning the repo. You're pulling in all of the Next.js source + you have to build it etc.